telegrambots-abilities/src/main/java/org/telegram/abilitybots/api/util/AbilityExtension.java

```diff
@@ -4,4 +4,17 @@
  * An interface to mark a class as an extension. Similar to when a method returns an Ability, it is added to the bot, a method which returns an AbilityExtension will add all Abilities or Replies from this Extension to the bot.
  */
 public interface AbilityExtension {
+
+    // https://github.com/rubenlagus/TelegramBots/issues/834
+    String DEFAULT = "default";
+
+    /**
+     * https://github.com/rubenlagus/TelegramBots/issues/834
+     * Method for accessing AbilityExtension.DEFAULT
+     * @return String "default"
+     */
+    default String getDefault() {
+        return AbilityExtension.DEFAULT;
+    }
+
```

Comment:
I don't like the idea that there's two declarations to this variable (here + in base). We can switch up the DEFAULT variable to become public. I don't see a problem in widening that.

Reply:
On a second thought, we can keep this here and remove the one in base ability bot. That class already extends AbilityExtension and people who're implementing the default outside the bot's class will benefit from this localization.

Also, ye this is definitely a static variable with a static method.